pkg/operator/controller/gatewayapi/controller.go

```diff
@@ -107,6 +133,12 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
+	if crdNames, err := r.listUnmanagedGatewayAPICRDs(ctx); err != nil {
+		return reconcile.Result{}, fmt.Errorf("failed to list unmanaged gateway CRDs: %w", err)
+	} else if err = r.setUnmanagedGatewayAPICRDNamesStatus(ctx, crdNames); err != nil {
+		return reconcile.Result{}, fmt.Errorf("failed to update the ingress cluster operator status: %w", err)
```

Comment:
```suggestion
		return reconcile.Result{}, fmt.Errorf("failed to update unmanaged gateway CRDs: %w", err)
```

Reply:
Wait, is that what `setUnmanagedGatewayAPICRDNamesStatus` is doing?  It looks like it is updating the clusteroperator status, not the CRDs.

Reply:
Okay.  Ignore this.